views/pages/services/student-services.pug

```diff
@@ -0,0 +1,54 @@
+extends /views/templates/frontend-base.pug
+include /views/mixins/pink-button.pug
+
+append styles
+    link(href="/assets/css/pages/services/student-services.css" rel="stylesheet")  
+    link(rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.2/css/all.min.css")
+
+block body 
+    div(class="service")
+        div(class="service-item")
+            h2 Locker Sign Out 
+            p If a student in the math faculty is on a study term and has paid their MathSoc fee, they are eligible to book a locker for the period of the term. No perishable foods are allowed to be stored to avoid attracting pests. MathSoc offers lockers on a term-by-term basis. Locker sign ups are closed for the term.
+        img(src="/assets/img/services/lockers.jpg", alt="lockers", class="service-item")
+    div(class="service reverse")
```

Comment:
Could you use `section` tags for each `div.service`?  It would help for accessibility!